tests/release/pipelines/e2e-test-default-with-deployment.go

```diff
@@ -13,10 +15,9 @@ import (
 	releaseApi "github.com/redhat-appstudio/release-service/api/v1alpha1"
 	tektonutils "github.com/redhat-appstudio/release-service/tekton/utils"
 	corev1 "k8s.io/api/core/v1"
-	"os"
 )
 
-var _ = framework.ReleaseSuiteDescribe("[HACBS-1199]test-release-e2e-with-deployment", Label("release-pipelines", "withDeployment", "HACBS"), func() {
+var _ = framework.ReleaseServicesSuiteDescribe("[HACBS-1199]test-release-e2e-with-deployment", Label("release-services", "withDeployment", "HACBS"), func() {
```

Comment:
why is this even being changed? It is in the `release/pipelines` dir, not a `service` dir. It is pretty confusing to me to have `release-service` tests in the `release/pipelines` dir... I would expect the `release/pipelines` dir to only have `release-pipelines` tests

Reply:
you are right @johnbieren , this change should be covered by https://github.com/redhat-appstudio/e2e-tests/pull/843. I restore the change.